frontend/src/lib/notify/index.ts

```diff
@@ -0,0 +1,38 @@
+import { writable } from 'svelte/store';
+export interface Notification {
+  id: string;
+  message: string;
+  category: string;
+  duration: number;
+}
+
+export const notifications = writable([]);
```

Comment:
Let's only expose our own notification functions for writing to the store.
```suggestion
const _notifications = writable([]);
export const notifications = readonly(_notifications);
```